rpmlint/checks/FileDigestCheck.py

```diff
@@ -444,7 +465,7 @@ def _check_symlink_exceptions(self, pkg, name):
         """
 
         for symlink_exception in self.symlink_exceptions:
-            if pkg.name not in self._gather_packages_from_dict(symlink_exception):
+            if self._matches_pkg(symlink_exception, pkg):
```

Comment:
Did you drop a `not` here? Looks to me like you're inverting the behaviour

Reply:
Good catch. I'll fix it.

Reply:
I force pushed a rebased commit that fixes this.

Reply:
Should this have been caught by a test, btw?

Reply:
Obviously not, since it isn't covered by the tests :-P

It **could have** been caught. There's never 100 % coverage I guess.